File: Workshop/JavaBasicsProject/src/Basics/Loops/CountFromTo.java
package Basics.Loops;

import java.util.Scanner;

public class CountFromTo {
  public static void main(String[] args) {
    // Create a program that asks for two numbers
    // If the second number is not bigger than the first one it should print:
    // "The second number should be bigger"
    //
    // If it is bigger it should count from the first number to the second by one
    //
    // example:
    //
    // first number: 3, second number: 6, should print:
    //
    // 3
    // 4
    // 5

    int from, to;
    Scanner input = new Scanner(System.in);
    boolean bigger = false;


    do {
      System.out.println("Tell me number to count from: ");
      from = input.nextInt();

      System.out.println("Tell me the number where counting should end: ");
      to = input.nextInt();

        if (to <= from) {
            System.out.println("The second number should be bigger");
        } else {
            bigger = true;
        }
    } while (!bigger);

    while (from <= to) {
      System.out.println(from);
      from++;
    }

  }


}

File: Foundation/week_02-week_04/PythonBasicsProject/week_02/expressions_and_control_flow/loops/count_from_to.py
# Create a program that asks for two numbers
# If the second number is not bigger than the first one it should print:
# "The second number should be bigger"
#
# If it is bigger it should count from the first number to the second by one
#
# example:
#
# first number: 3, second number: 6, should print:
#
# 3
# 4
# 5
a = -999
b = -1999
while a > b:
    if a != -999:
        print("\nthe second number should be bigger")
    a = int(input("gimme a number to count from: "))
    b = int(input("gimme a number to count to: "))

while a < b:
    print(a)
    a += 1

# for x in range(b):
#     if x < a:
#         continue
#     print(x)
